session/http_server.go

```diff
@@ -168,6 +168,7 @@ func encodeCookieSession(w http.ResponseWriter, s *influxdb.Session) {
 	c := &http.Cookie{
 		Name:  cookieSessionName,
 		Value: s.Key,
+		Path:  "/api/",
 	}
```

Comment:
This path needs to be specified for the session cookie because the endpoint for notebooks is prefixed with `/api/v2private/flows`. Without specifying a path, a browser (firefox, at least) will default to a path of `/api/v2/` since the signin url is prefixed with `/api/v2/`, which causes the session to not be valid for paths prefixed with `/api/v2private/flows`.